tests/mcs_benchmark_data_test/pipelines/CommonsenseQA/kagnet_commonsense_qa_submission_transformer_test.py

```diff
@@ -6,15 +6,15 @@
     KagnetCommonsenseQaSubmissionFileNames,
 )
 
-from tests.assertions import assert_submission_models
+from tests.mcs_benchmark_data_test.assertions import assert_submission_models
 
 
 def test_extract_transform():
     models = tuple(
         KagnetCommonsenseQaSubmissionPipeline(
             file_names=KagnetCommonsenseQaSubmissionFileNames(
-                metadata="CommonsenseQA_dev_submissions.jsonl",
-                submission="dev_rand_split_kagnet_submission.jsonl",
+                metadata="submissions_metadata.jsonl",
+                submission="dev_rand_split_kagnet_submission_small.jsonl",
```

Comment:
The _small suffix will go away, since you'll have the same file names and subdirectories in `data/` and `test_data/`, just with different data (`test_data/` with the current smaller files).